frontend/src/app/components/pay-page/pay-page.component.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
- [ ] These tests are too trivial, either add more thorough tests or just remove this test file

Reply:
Actually since all these components are pretty simple, I don't think we should invest in tests for the Frontend